datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/useGetEntitySchema.tsx

```diff
@@ -0,0 +1,24 @@
+import { cloneDeep } from 'lodash';
+import { EntityType } from '../../../../../../types.generated';
+import { useEntityData } from '../../../EntityContext';
+import { useGetDatasetSchemaQuery } from '../../../../../../graphql/dataset.generated';
+import { combineEntityDataWithSiblings, useIsSeparateSiblingsMode } from '../../../siblingUtils';
+
+export const useGetEntityWithSchema = () => {
+    const { urn, entityData, entityType } = useEntityData();
+    // Load the dataset schema lazily.
+    const { data: rawData, loading } = useGetDatasetSchemaQuery({
+        variables: {
+            urn,
+        },
+        skip: entityType !== EntityType.Dataset,
+        fetchPolicy: 'cache-first',
```

Comment:
is editing schema field documentation, tags, and terms affected here with `cache-first`?

Reply:
i've actually updated to address!